Nailgun servers fail to start with JDKs > 12.

Trying with e.g. `jdk="adopt:1.13"`:
```
21:40:09.09 [ERROR] 1 Exception encountered:

  Exception: Error communicating with server: invalid digit found in string

```

This is likely just because there is some warning output on startup, which we are parsing rather than the port number.

--- a/src/rust/engine/nailgun/Cargo.toml
+++ b/src/rust/engine/nailgun/Cargo.toml
@@ -10,8 +10,7 @@ async_latch = { path = "../async_latch" }
 bytes = "1.0"
 futures = "0.3"
 log = "0.4"
-# TODO: Stabilize before landing.
-nails = { git = "https://github.com/stuhood/nails.git", rev = "aee6120e2688488bc519dfedb77cc05f704cbc89" }
+nails = "0.13"
 os_pipe = "1.0"
 task_executor = { path = "../task_executor" }
 tokio = { version = "1.16", features = ["fs", "io-std", "io-util", "net", "signal", "sync"] }

--- a/src/rust/engine/process_execution/Cargo.toml
+++ b/src/rust/engine/process_execution/Cargo.toml
@@ -22,8 +22,7 @@ futures = "0.3"
 hashing = { path = "../hashing" }
 libc = "0.2.126"
 log = "0.4"
-# TODO: Stabilize before landing.
-nails = { git = "https://github.com/stuhood/nails.git", rev = "aee6120e2688488bc519dfedb77cc05f704cbc89" }
+nails = "0.13"
 nix = "0.24"
 sha2 = "0.10"
 shell-quote = "0.3.0"

